README.md

```diff
@@ -8,6 +8,12 @@ This repo contains Ansible code for OpenShift and Atomic Enterprise.
   ```
     dnf install -y ansible-1.9.4 rubygem-thor rubygem-parseconfig util-linux pyOpenSSL libffi-devel python-cryptography
   ```
+  - RHEL 7:
+  ```
+  yum install http://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+  rpm --import http://dl.fedoraproject.org/pub/epel/RPM-GPG-KEY-EPEL-7
+  yum install ansible1.9-1.9.4-2.el7 rubygem-thor rubygem-parseconfig util-linux pyOpenSSL libffi-devel python-cryptography
```

Comment:
Ok, so it won't be a complete guide for RHEL (like it is for Fedora), but only a note, something like: "For RHEL, install epel and in the Fedora list replace ansible-1.9.4 with ansible1.9-1.9.4-2.el7 ". It might be little confusing for the reader. Any phrasing you choose, is good, as long as ppl will have at least some hints how to proceed.